deploy/infrastructure/modules/terraform-aws-dss/README.md

```diff
@@ -0,0 +1,74 @@
+   
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named, for instance, `terraform-aws-dss-dev`.
+2. Copy main.tf, output.tf and variables.tf to the new folder.
+3. Copy `terraform.dev.example.tfvars` and rename to `terraform.tfvars`
+4. Check that your new directory contains the following files:
+   - main.tf
+   - output.tf
+   - terraform.tfvars
+   - variables.tf
+5. Set the variables in `terraform.tfvars` according to your environment. See [TFVARS.md](TFVARS.md) for variables descriptions.
+6. In the new directory (ie /deploy/infrastructure/personal/terraform-aws-dss-dev), initialize terraform: `terraform init`.
+7. Run `terraform plan` to check that the configuration is valid. It will display the resources which will be provisioned.
+8. Run `terraform apply` to deploy the cluster. (This operation may take up to 15 min.)
+9. If `aws_route53_zone_id` was not provided, configure the DNS resolution to the public ip addresses and for SSL certificate validation.  
+See [DNS](DNS.md) for detailed instructions for AWS Zone ID setup (DNS managed with terraform) or manual configuration. 
+
+
+## Deployment of the DSS services
+
+During the successful run, the terraform job has created a new [workspace](../../../../build/workspace/)
+for the cluster. The new workspace name corresponds to the cluster context. The cluster context
+can be retrieved by running `terraform output` in your infrastructure  folder (ie /deploy/infrastructure/personal/terraform-aws-dss-dev).
+
+It contains scripts to operate the cluster and setup the services.
+
+1. Go to the new workspace `/build/workspace/${cluster_context}`.
+2. Run `./get-credentials.sh` to login to kubernetes. You can now access the cluster with `kubectl`.
+3. Generate the certificates using `./make-certs.sh`. Follow script instructions if you are not initializing the cluster.
+4. Deploy the certificates using `./apply-certs.sh`.
+5. Run `tk apply .` to deploy the services to kubernetes. (This may take up to 30 min)
+6. Wait for services to initialize:
+    - On AWS, load balancers and certificates are created by Kubernetes Operators. Therefore, it may take few minutes (~5min) to get the services up and running and generate the certificate. To track this progress, go to the following pages and check that:
+        - On the [EKS page](https://eu-west-1.console.aws.amazon.com/eks/home), the status of the kubernetes cluster should be `Active`.
+        - On the [EC2 page](https://eu-west-1.console.aws.amazon.com/ec2/home#LoadBalancers:), the load balancers (1 for the gateway, 1 per cockroach nodes) are in the state `Active`.
+7. Verify that basic services are functioning by navigating to https://your-gateway-domain.com/healthy.
+
+
+## Clean up
+
+1. Note that the following operations can't be reverted and all data will be lost.
+2. To delete all resources, run `tk destroy .` in the workspace folder.
```

Comment:
There is no command called `tk destroy .` , did you mean `tk delete .` ? 
![image](https://user-images.githubusercontent.com/1588744/216954660-62975fa4-fe76-4162-a90a-dd4a43aa6aa9.png)

Reply:
For some reason, tanka is generating an error when I do `tk delete .`

![image](https://user-images.githubusercontent.com/1588744/216956077-7aabadfb-ad52-4ce2-9d26-db55c756c401.png)

Reply:
The deletion process works its just that tanka gives a error.

Reply:
Thanks, fixed in the documentation